ui/src/websocket-client.ts

```diff
@@ -59,6 +59,8 @@ export type WebSocketActionParams = AnyObject | AnyObject[];
 
 export type WebSocketAction<P = WebSocketActionParams> = PayloadAction<
   {
+    formId?: string;
+    modelPk?: string;
     params: P;
   },
```

Comment:
As mentioned above I think it makes more sense for this data to be part of `payload` itself, not `meta`.